frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -205,6 +206,25 @@ export const PublicFormProvider = ({
         return showErrorToast(error, form)
       }
 
+      const countryRegionFieldIds = new Set(
+        form.form_fields
+          .filter((field) => field.fieldType == BasicField.CountryRegion)
+          .map((field) => field._id),
+      )
+      const formInputsWithCountryRegionInUpperCase = Object.keys(
```

Comment:
can consider using `newFormInputs: typeof formInputs` instead of `any`

Reply:
TIL. Will do. Thank you for helping me get rid of `any`.